web/src/styles/globals.css

```diff
@@ -0,0 +1,72 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@font-face {
+  font-family: "Chalkboard SE";
```

Comment:
instead of doing this here, let's use the nextjs font package: https://nextjs.org/docs/basic-features/font-optimization#local-fonts

Reply:
Alright, I'll move it there